src/main/java/frc/robot/subsystems/DriveSubsystem.java

```diff
@@ -272,8 +241,10 @@ public void smartMotionToPosition(double left, double right, boolean turning) {
     m_drive.feed();
     if (turning) {
       int pidSlot = 1;
-      m_rightController.setReference(right, ControlType.kSmartMotion, pidSlot, 0.04, ArbFFUnits.kPercentOut);
-      m_leftController.setReference(left, ControlType.kSmartMotion, pidSlot, 0.04, ArbFFUnits.kPercentOut);
+      m_rightController.setReference(
+          right, ControlType.kSmartMotion, pidSlot, 0.04, ArbFFUnits.kPercentOut);
```

Comment:
Ahh, this .04 is where the magic static feed forward offset is inputted. We should definitely make a variable in Constants out of this and use it especially for turning.

Reply:
0.04 also wasn't a value that worked, although we should still have the old value somewhere. I'll make sure to test that tomorrow :)